nextflow.config

```diff
@@ -65,11 +65,14 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
-    // Extract Kraken2 Reads
+    // Extract kraken2 Reads
     extract_kraken2_reads            = true
+    fastq_output                     = false
```

Comment:
Shall fastq format be the default so that we are consistent across classifiers?

Reply:
Agree!